internal/pkg/aws/ecs/ecs.go

```diff
@@ -134,6 +138,38 @@ func (e *ECS) listTasks(cluster string, opts ...listTasksOpts) ([]*Task, error)
 	return tasks, nil
 }
 
+// StopTasksOpts sets the optional parameter for StopTasks.
+type StopTasksOpts func(*ecs.StopTaskInput)
+
+// WithStopTaskReason sets an optional message specified when a task is stopped.
+func WithStopTaskReason(reason string) StopTasksOpts {
+	return func(in *ecs.StopTaskInput) {
+		in.Reason = aws.String(reason)
+	}
+}
+
+// WithStopTaskCluster sets the cluster that hosts the task to stop.
+func WithStopTaskCluster(cluster string) StopTasksOpts {
+	return func(in *ecs.StopTaskInput) {
+		in.Cluster = aws.String(cluster)
+	}
+}
+
+// StopTasks stops multiple running tasks.
+func (e *ECS) StopTasks(tasks []string, opts ...StopTasksOpts) error {
```

Comment:
Is this the ARN or the ID of the tasks? Could be helpful for future programmers to specify that in the names of the arguments.
```suggestion
func (e *ECS) StopTasks(taskARNs []string, opts ...StopTasksOpts) error {
```

Reply:
It could be ARN or ID. In my PR I pass in the ARN but this public function itself, the parameter could accept both task IDs or ARNs.

Reply:
Gotcha, I think we should call it out in the comment since this is a public method.